setup.sh

```diff
@@ -55,10 +55,12 @@ main () {
   # Check that the user is using a supported notifier type in the correct
   # directory.
   case "${NOTIFIER_TYPE}" in
-  http|smtp|slack) ;;
+  http|smtp|slack|bigquery) ;;
   *) fail "${HELP}"
   esac
 
+  echo "${SOURCE_CONFIG_PATH}"
```

Comment:
Remove unless you feel like this would useful here (I'm open to any usability suggestions with this script).

Reply:
Resolved!